### PR TITLE
Protect from NPE when paths aren't provided

### PR DESCRIPTION
**Pull request by agilob**
### PR checklist

- [x] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [x] If contributing template-only or documentation-only changes which will change sample output, [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) beforehand.
- [x] Run the shell script `./bin/generate-samples.sh`to update all Petstore samples related to your fix. This is important, as CI jobs will verify _all_ generator outputs of your HEAD commit as it would merge with master. These must match the expectations made by your contribution. You may regenerate an individual generator by passing the relevant config(s) as an argument to the script, for example `./bin/generate-samples.sh bin/config/java*`. For Windows users, please run the script in [Git BASH](https://gitforwindows.org/).
- [x] File the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`
- [x] Copy the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) to review the pull request if your PR is targeting a particular programming language.

fixes https://github.com/OpenAPITools/openapi-generator/issues/6733

### CODE REVIEW

**Comment by wing328** [comment]
cc @OpenAPITools/generator-core-team

File: modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultGenerator.java [hunk 1]
```diff
@@ -892,7 +892,7 @@ private void generateSupportingFiles(List<File> files, Map<String, Object> bundl
 
             sb.append(System.lineSeparator());
 
-            System.err.println(sb.toString());
+            LOGGER.debug(sb.toString());
```
Comment (wing328) [inline modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultGenerator.java:895]: @agilob thanks for the PR. I no longer get the NPE using the spec you provided 👍 

For this line, what about using `LOGGER.error` instead?

Comment (agilob) [reply modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultGenerator.java]: replaced

**Comment by wing328** [comment]
Travis CI failure already fixed in the master.

**Comment by wing328** [comment]
@agilob thanks for the PR, which has been merged into master.

**Pull request merged**